pkg/services/cleanup/cleanup.go

```diff
@@ -74,7 +74,7 @@ func (j cleanUpJob) String() string {
 func (srv *CleanUpService) Run(ctx context.Context) error {
 	srv.cleanUpTmpFiles(ctx)
 
-	ticker := time.NewTicker(time.Minute * 10)
+	ticker := time.NewTicker(time.Minute * 1)
```

Comment:
This is done purely for ephemeral instance purposes, so we can observe the cleanup happening more often in metrics.

@alexweav: revert this line before submission.